KakapoExample/KakapoTests/KakapoDBTests.swift

```diff
@@ -277,6 +437,56 @@ class KakapoDBTests: QuickSpec {
                 }
                 expect(user).toEventuallyNot(beNil())
             }
```

Comment:
I think this test might be pointless because there is no way to make them fail, delete and update are not executing "external" code